data-catalog/base/thriftserver/postgres-service.yaml

```diff
@@ -0,0 +1,19 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: postgres
+  annotations:
```

Comment:
When generated, this file is missing a label `template: dh-data-catalog`

Reply:
Do you want that label to be applied on all manifests in data catalog or on just these two instances? We can do that via the `commonLabels` in `kustomization.yaml` for all the resources. :wink: